livechat/utils/ws_client.py

```diff
@@ -2,153 +2,96 @@
 Client for WebSocket connections.
 '''
 
-# pylint: disable=R1702
-
 import json
 import logging
 import random
 import ssl
 import threading
 from time import sleep
-from timeit import default_timer as timer
+from typing import List
+
+from websocket import WebSocketApp, WebSocketConnectionClosedException
+from websocket._abnf import ABNF
+
+
+def on_message(ws_client: WebSocketApp, message: str):
+    ''' Custom WebSocketApp handler that inserts new messages in front of `self.messages` list. '''
+    ws_client.messages.insert(0, json.loads(message))
```

Comment:
Why is it outside of the WebsocketClient?

Reply:
it can be either inside init or outside the class, it won't work if I just overwrite the function inside class. Don't ask why exactly, because I'm not the author of WebSocketApp class, but I assume it has something to do with the configuration going on inside its' init function